helm/aladdin-demo/templates/commands/deploy.yaml

```diff
@@ -41,13 +41,4 @@ spec:
         ports:
         - containerPort: {{ .Values.app.uwsgi.port }}
           protocol: TCP
-{{ if .Values.deploy.withMount }}
-        volumeMounts:
-        - mountPath: /code/commands
-          name: {{ .Chart.Name }}-commands
-          subPath: app/commands_app/commands
-      volumes:
-      - name: {{ .Chart.Name }}-commands
-        persistentVolumeClaim:
-          claimName: {{ .Chart.Name }}-nfs-volume-claim
-{{ end }}
```

Comment:
no more mounting in the commands pod?

Reply:
Let me take a closer look.

Reply:
Urgh.  I reverted this earlier commit without looking closely enough.  I'll fix this.